manifests/release/kubevirt.yaml.in

```diff
@@ -242,3 +243,35 @@ spec:
         - name: virt-private-dir
           hostPath:
             path: /var/run/kubevirt-private
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: virtualmachinepresets.kubevirt.io
+  labels:
+    kubevirt.io: ""
+spec:
+  group: kubevirt.io
+  version: v1alpha1
+  scope: Namespaced
+  names:
+    plural: virtualmachinepresets
+    singular: virtualmachinepreset
+    kind: VirtualMachinePreset
+    shortNames:
+      - vmpreset
+      - vmpresets
+---
+apiVersion: admissionregistration.k8s.io/v1alpha1
```

Comment:
We either need an extra manifest for the InitializerConfiguration, or the controller will need to do this configuration programmatically and consider the possibility that this alpha feature is not enabled on the cluster. Otherwise we break deployment on most already existing clusters.

Reply:
Yikes! You are correct. Shipping this manifest as-is will indeed trigger an error just trying to deploy this.